### PR TITLE
Add `set_select` to set attr value to a `select`

### PR DESCRIPTION
**Pull request by healthy-pod**
This code change adds a new command `set_select` which allows setting the value of an attribute to a call to `select` and builds the dict using the KVs passed to it.

Example call:
```
set_select <attr> <key_1> <value_1> <key_n> <value_n>
```

### CODE REVIEW

**Comment by healthy-pod** [comment]
Ready for review.

**Comment by healthy-pod** [comment]
@vladmos @pmbethe09 @meisterT can you please take a look?

**Review by vladmos — APPROVED** [review]

File: edit/buildozer_test.go [hunk 1]
```diff
@@ -439,3 +439,41 @@ func TestCmdDictAddSet_missingColon(t *testing.T) {
 		})
 	}
 }
+
+func TestCmdSetSelect(t *testing.T) {
+	buildFile := `foo(
+		name = "foo",
+	)`
```
[the remark this answers is not in the document]
Comment (healthy-pod) [reply edit/buildozer_test.go]: Thanks for the review, ready for another look.

[the remark this answers is not in the document]
Comment (healthy-pod) [reply edit/buildozer_test.go]: @vladmos friendly ping :)

**Pull request merged**